sdk/core/azure-core/test/ut/requirements.txt

```diff
@@ -0,0 +1 @@
+websockets
```

Comment:
What's the purpose of this file?

Reply:
Oh, I see now, it's for `pip install -r requirements.txt`

Consider renaming the file to be `package_requirements.txt`.

Reply:
Everywhere else I've seen this file called "requirements.txt". That is the convention for all python components.

Do you have a strong reason for what I suspect is a non-pythonic change?